package.json

```diff
@@ -15,6 +15,7 @@
     "dist": "yarn build dist",
     "build": "yarn lavamoat:build",
     "build:dev": "node development/build/index.js",
+    "build:icons": "node ui/components/component-library/icon/scripts/generate-icon-names.js && node ui/components/component-library/icon/scripts/generate-icon-scss.js",
```

Comment:
Seems that this was left in by mistake? `generate-icon-scss.js` doesn't exist anymore, and `generate-icon-names.js` is now a library rather than a CLI.